### DISCUSSION TITLE
Zeek OTX source/desc

### DISCUSSION THREAD

**Discussion by pzafar1974** [Security-Onion-Solutions/securityonion#2770] [General]
HI,
I downloaded Zeek-OTX script and installed it is working fine. What I have noticed in the intel.log it is logging the meta. source which has only the author name of the feed. I would like to see the meta.desc In the intel.log as it was in the older script I use from the other source. 

How can I change this? I would appreciate any help.

Thanks

**Comment by weslambert** [comment]
I'll have to look into this and get back with you, as TMK, nothing with the intel.dat itself should have change.
